<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wEditSeason</class>
 <widget class="QMainWindow" name="wEditSeason">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wEditSeason</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>The default season is summer and does not need to be defined.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="lblStart">
        <property name="text">
         <string>Start:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblEnd">
        <property name="text">
         <string>End:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblSeason">
        <property name="text">
         <string>Season:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cobxSeason">
        <item>
         <property name="text">
          <string>Summer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spring</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Autumn</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Winter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Deep winter</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="daeEnd">
        <property name="displayFormat">
         <string>dd.MM.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="daeStart">
        <property name="displayFormat">
         <string>dd.MM.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnDiscard">
        <property name="text">
         <string>Discard changes</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
